LeezeIt-Inc/enterprise
# [KAS-16] Make Leezeit tutoruial page

### Jira Story Link:
[KAS-16](https://leezeit.atlassian.net/browse/KAS-16?atlOrigin=eyJpIjoiMDJjYTdlOTcyNmQ1NDQ3YmI0N2I0MjBkNGEwOGQ2NWYiLCJwIjoiaiJ9)

**Description:** 
1)   Build swipe-able  tutorial pages.
2)  Tutorial consist of two pages.
3)  Dot-1 is selected on first page and Dot-2 is selected on second page.
4)  There is next button on Second page to go to LogIn/SignUp flow.
5)  Tutorial pages only show when user open app first time after installing.

**How to test this:**
Run the app first time after installation  to see tutorial pages.
Run again to see log In Sign Up flow.

**ScreenShots & Gifs:**


https://user-images.githubusercontent.com/111387186/190896354-d98fc060-4486-4cbe-9932-e68e80059e03.mp4
